src/pages/User.js

```diff
@@ -0,0 +1,44 @@
+import logo from '../CodeKingTrans.png';
+import Navbar from '../components/Navbar.js';
+import Footer from '../components/Footer.js';
+import List from '../components/List.js';
+import useFetch from '../hooks/useFetch';
+import '../App.css';
+
+function UserCard({ user }){
+    return (
+        <div className='userCard'>
+            <img alt='profile logo' src='https://cdn-icons-png.flaticon.com/512/12225/12225773.png'></img>
+            <h2>{user.firstName + ' ' + user.lastName}</h2>
+        </div>
+    );
+}
+
+
+function User() {
+    const { data: users, loading, error } = useFetch("https://disc-assignment-5-users-api.onrender.com/api/users");
+    
+
+
+    return (
+        <div className="App">
+            <Navbar />
+            <div className='backdrop'>
+                <img alt='Code King Logo' src={logo}></img>
+                <div className="userList">
+                    <div className='userListTitle'>
+                        <h1>Friends:</h1>
+                    </div>
+                    {loading && <p>Loading...</p>}
+                    {error && <p>Error: {error.message}</p>}
+                    {users && (
+                        <List items={users} renderItem={(user) => <UserCard key={user.id} user={user} />} />
+                    )}
```

Comment:
While this pattern is fine, I would recommend a slightly different pattern that enforces the fact that you only ever want to have ONE of [loading, error, or users] being displayed at a time.
If you were to have a bug with your fetch (or the backend api has a bug with its response), it could be possible that loading is true and error is true and users are true (or some combination of the three that shouldnt be allowed). So just to add extra error protection, I would make this an if/else statement so only one of these is every displayed at a time.